src/funchook.c

```diff
@@ -170,6 +170,11 @@ int funchook_destroy(funchook_t *funchook)
     return rv;
 }
 
+const char *funchook_error_message(const funchook_t *funchook)
+{
+    return ((struct funchook *)funchook)->error_message;
```

Comment:
Thanks! I forgot to implement it.
Could you change this line to `return funchook->error_message;`?

Reply:
Whoops, that was silly of me. Done.